src/db/prisma/schema.prisma

```diff
@@ -0,0 +1,72 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Job {
+  id              String          @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  data            Json            @default("{}")
+  status          OperationStatus @default(PENDING)
+  percentage      Int             @default(0)
+  creation_time   DateTime?       @default(now())
+  update_time     DateTime?       @updatedAt
+  expiration_time DateTime?
+  type            JobMode         @default(PREDEFINED)
+  metadata        Json?
```

Comment:
dont you want it to be the same as data? (default {} and always defined)

Reply:
changed. now it act like data